easybuild/framework/easyblock.py

```diff
@@ -235,8 +243,12 @@ def _init_log(self):
         self.log.info(this_is_easybuild())
 
         this_module = inspect.getmodule(self)
-        self.log.info("This is easyblock %s from module %s (%s)",
-                      self.__class__.__name__, this_module.__name__, this_module.__file__)
+        eb_class = self.__class__.__name__
+        eb_loc = this_module.__file__
+        self.log.info("This is easyblock %s from module %s (%s)", eb_class, this_module.__name__, eb_loc)
+
+        if self.dry_run:
+            dry_run_msg("*** DRY RUN using '%s' easyblock (from %s) ***\n" % (eb_class, eb_loc), silent=self.silent)
```

Comment:
this is really nitpicking but: `DRY RUN using easyblock '%s' `

And why not print the module name in case of dry run?